vmm/src/device_manager.rs

```diff
@@ -895,6 +895,7 @@ struct MetaVirtioDevice {
     iommu: bool,
     id: String,
     pci_segment: u16,
+    pci_device_hint: Option<u16>,
```

Comment:
would `pci_bdf_hint` or `pci_addr_hint` be a better name? Or can we drop the "hint" at all? It is not an hint. We either get the requested PCI address or we fail